<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WBalanceI</class>
 <widget class="QWidget" name="WBalanceI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>175</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>175</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>175</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(128, 128, 128, 0);</string>
  </property>
  <widget class="WLabel" name="lbtbvohms">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>128</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Centro</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvbU">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>80</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string>%</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Izquierda</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvselT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>48</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string>TBVSEL</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Derecha</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvselU">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>48</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string>%</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Izquierda</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvsel">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>48</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Centro</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvma">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>112</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Centro</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbva">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>64</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Centro</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvmaT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>112</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string>TBVMA</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Derecha</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvmaU">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>112</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string>MA</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Izquierda</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvdmdT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>32</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string>TBVDMD</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Derecha</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvohmsT">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>128</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string>TBVOHMS</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Derecha</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvohmsU">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>128</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string>OHMS</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Izquierda</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvdmdU">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>32</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string>%</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Izquierda</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvb">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>80</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Centro</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvdmd">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>32</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Centro</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvaU">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>64</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string>%</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Izquierda</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvaT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>64</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string>TBVA</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Derecha</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbtbvbT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string>TBVB</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Derecha</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btCerrar">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>0</y>
     <width>173</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 192, 192);</string>
   </property>
   <property name="text">
    <string>BALANCE IMPULSO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>175</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>lbtbvohms</zorder>
  <zorder>lbtbvbU</zorder>
  <zorder>lbtbvselT</zorder>
  <zorder>lbtbvselU</zorder>
  <zorder>lbtbvsel</zorder>
  <zorder>lbtbvma</zorder>
  <zorder>lbtbva</zorder>
  <zorder>lbtbvmaT</zorder>
  <zorder>lbtbvmaU</zorder>
  <zorder>lbtbvdmdT</zorder>
  <zorder>lbtbvohmsT</zorder>
  <zorder>lbtbvohmsU</zorder>
  <zorder>lbtbvdmdU</zorder>
  <zorder>lbtbvb</zorder>
  <zorder>lbtbvdmd</zorder>
  <zorder>lbtbvaU</zorder>
  <zorder>lbtbvaT</zorder>
  <zorder>lbtbvbT</zorder>
  <zorder>btCerrar</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WLabel</class>
   <extends>QWidget</extends>
   <header>wlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
